docs/events/timeline-ui-overview.asciidoc

```diff
@@ -2,84 +2,72 @@
 = Investigate events in Timeline
 
 Use Timeline as your workspace for investigations and threat hunting.
-Data from multiple indices can be added to a Timeline, which enables
-investigating complex threats.
+You can add data from multiple indices to a Timeline to facilitate advanced investigations.
 
-You can drag or send items of interest to Timeline to create the query you need
-to investigate an alert or event. You can add items from tables and histograms
+You can drag or send items of interest to a Timeline to create the desired query. You can add items from tables and histograms
 on the *Overview*, *Alerts*, *Hosts*, and *Network* pages, as well as from
-within a Timeline itself. Additionally, you can add a query directly in Timeline
-(click *+ Add field*).
+other Timelines. Additionally, you can add a query directly in Timeline
+by clicking *+ Add field*.
 
 [role="screenshot"]
 image::images/timeline-ui-updated.png[]
 
 Timelines are responsive, and they persist as you move through the {es-sec-app}
 collecting data. Auto-saving ensures that the results of your investigation are
-available for review by other analysts and incident response teams. To record
-and share your findings with others, attach your Timeline to a
-<<cases-overview, case>>.
+available for later review. To record and share your findings with others, attach your Timeline to a <<cases-overview, case>>.
```

Comment:
We link to the Cases topic here, but we don't actually have an area in that topic that describes how to do that. I can file a doc issue to create a section so we can update this link and send users to something that's more specific and relevant.